### DISCUSSION TITLE
What is this colorscheme for vim?

### DISCUSSION THREAD

**Discussion by mannaggiaAJesu** [w0rp/ale#4570] [Q&A - Ask for help with problems]
![60982985-613d2e00-a363-11e9-97c1-b59dc80ec998](https://github.com/dense-analysis/ale/assets/140708771/f4221eb6-7d4e-4e4a-b66c-06330ebc103b)
Someone know the name of this colorscheme?

**Comment by mannaggiaAJesu** [comment]
I want the highlights of this theme and I think is gruvbox but I don't know if i a folk of the colorscheme

**Reply by w0rp** [reply]
Probably better to ask this kind of question in /r/vim on Reddit.
